net/http/http_stream_factory_job_controller.cc

```diff
@@ -817,10 +820,23 @@ int HttpStreamFactory::JobController::DoCreateJobs() {
         GetAlternativeServiceInfoFor(request_info_, delegate_, stream_type_);
   }
   quic::ParsedQuicVersion quic_version = quic::ParsedQuicVersion::Unsupported();
+#if defined(STARBOARD)
+  bool protocol_filter_override = false;
+#endif  // defined(STARBOARD)
   if (alternative_service_info_.protocol() == kProtoQUIC) {
-    quic_version =
-        SelectQuicVersion(alternative_service_info_.advertised_versions());
+#if defined(STARBOARD)
+    if (alternative_service_info_.protocol_filter_override() &&
```

Comment:
I'm trying to understand the need to add protocol_filter_override—if we instead changed `DefaultSupportedQuicVersions` to include Q046 as well wouldn't we not need that variable at all? It seems to me that would save us a lot of customizations to //net